docs/environment/custom-domains.md

```diff
@@ -31,9 +31,10 @@ After validating the domain and the certificate we can now link the custom domai
 - click "Create"
 - enter your domain name, select the certificate you created above and save
 - edit the domain that was created
-- add a "Path mapping" to map the `/` URL (or any URL you want) to your HTTP application in the `Prod` stage, for example:
+- click "Configure API mappings" to add an "API mapping" which maps the `/` URL (or any URL you want) to your HTTP application in the `Dev` stage, for example:
 
   ![](custom-domains-path-mapping.png)
+If you are using Cloudfront:
```

Comment:
```suggestion
If your endpoint type is `Edge`:
```

Reply:
This line needs to be removed even with regional domains. The DNS entry must be created.

Reply:
Ok, I'll remove it - but I had to create my CNAME entry and an AWS certificate before I was even able to get to this screen.